CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+### citus v10.2.4 (February 1, 2022) ###
+
+* Adds support for operator class parameters in indexes
+
+* Fixes a build error that happens when `lz4` is not installed
+
+* Fixes function signature generation bug
+
+* Improves self-deadlock prevention for `CREATE INDEX`, `REINDEX CONCURRENTLY`
```

Comment:
my bad;

```suggestion
* Improves self-deadlock prevention for `CREATE INDEX` & `REINDEX CONCURRENTLY` commands
```

Reply:
```md
* Improves self-deadlock prevention for `CREATE INDEX` &
  `REINDEX CONCURRENTLY` commands for builds using PG14 or higher
```

How does this sound?

Having an extra space before `&` breaks character limits for this line